text_extensions_for_pandas/array/arrow_conversion.py

```diff
@@ -139,13 +139,13 @@ def char_span_to_arrow(char_span: CharSpanArray) -> pa.ExtensionArray:
     return pa.ExtensionArray.from_storage(typ, storage)
 
 
-def arrow_to_char_span(extension_array: pa.ExtensionArray) -> CharSpanArray:
+def arrow_to_char_span(extension_array: pa.ExtensionArray) -> SpanArray:
```

Comment:
change function name to `arrow_to_span` ?

Reply:
I can do this in followup as well

Reply:
Good catch! I missed that while doing search-and-replace. Fixed now.